src/iceberg-comb/iceberg_comb.sh.in

```diff
@@ -321,6 +321,14 @@ then
     exit $EXIT_FAILURE
 fi
 
+# If all the iceberg files are empty, then there is no work to do.
+# Report this to the user and exit 0.
+if [ -z "$combine_files" ]
```

Comment:
This will always be true as `$combine_files` hasn't yet been defined.  

Please describe _why_ you want this option here?  If the number of files passed to this is zero, then it should be a failure.

Reply:
Thank you both for your time and feedback. I let the months slip by but we still need a fix to handle cases where there all the input icebergs are empty, so I'm going to finish this off now.

@underwoo , you're completely right that `$combine_files` is not yet defined here. I meant to put it right after the `has_icebergs` check on line 338.

@mlee03 line 318 is checking for the output file before generating it, and will exit with error if the output file already exists. So that's a different scenario than this special case where the input icebergs are empty.

If all the input icebergs are empty, the output file should be empty too. (It would be misleading to have a missing iceberg restart file when in fact the restart file is there but there are no icebergs).